tests/integration/conftest.py

```diff
@@ -1553,7 +1554,7 @@ async def submit(payload: BaseTaskPayload, **kwargs):
                     status=status,
                     result="",
                     children=[],
-                    date_done=datetime.utcnow(),
+                    date_done=datetime.now(pytz.timezone('UTC')),
```

Comment:
I have run into strange offset values with `pytz`. You may want to use `dateutil.UTZ` instead to be safe.